.jpb/persistence-units.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="PersistenceUnitSettings">
+    <persistence-units>
+      <persistence-unit name="Default">
+        <packages>
+          <package value="tahafurkan.sandbox.familybudgetapp" />
+        </packages>
+      </persistence-unit>
+    </persistence-units>
+  </component>
+</project>
```

Comment:
Bu dosyaya neden gerek var, olmayinca ne oluyor?

Reply:
bu dosyayı aslında ben oluşturmadım yaptıgım değişikliklerden dolayı kendisi oluşturuldu gerekli diye düşündüğüm için kaldırmadım.

Reply:
`JPA Buddy` diye bir plugin var IntelliJ'de, onu mu kullaniyorsun?

Reply:
evet